test-vectors/crypto-basics.json

```diff
@@ -0,0 +1,218 @@
+[
+  {
+    "cipher_suite": 1,
+    "derive_secret": {
+      "label": "DeriveSecret",
+      "out": "1bb35ac6bba0e621ddc0ad028e7cc87efbb9097f42b58b9ced0eb30f864c7fb9",
+      "secret": "2cbf39398db0dc4d7d123ed2c96b69d14c1c80bb50d9d88559a3acfd1fb5b789"
+    },
+    "encrypt_with_label": {
+      "ciphertext": "38dad2fc12bb3e911ddfc93f71455a68bdf2899616f426e9415fbc38f75222bc4b269f7cfc1d14abde766a35f8a7bfdf",
+      "context": "227f29e96cd9dd3d24691f04be7a36cb4f35611467323bd34901b6c13539a4c9",
+      "kem_output": "2fd428817a46920415a98bdadd759c8e80c6277bdaba5c60c8f7a76f3b630467",
+      "label": "EncryptWithLabel",
+      "plaintext": "1a3128a8280e98f1262c66097441499e7febb1ae3f7a2a5c260628a851886ed9",
+      "priv": "2062ea6092eae6d256a449111acb2d8ed022c7d03832a12e64e1adbe0df8e04c6b20b7d80118bc5a49f940866acf132e2b8a91205e2522113e2a9364107103497866",
+      "pub": "20b7d80118bc5a49f940866acf132e2b8a91205e2522113e2a9364107103497866"
```

Comment:
It's not challenging at all. But maybe it's worth mentioning in the document that this is done? It's a bit unintuitive since other values like ciphertext (which we also send on the wire) are not serialized.